docker/install_packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+. /build/install_pecan_preprocessor.sh
+
+echo "######################################################################"
+echo "SETTING UP REPOS"
+echo "######################################################################"
+case "$OS_VERSION" in
```

Comment:
This is copied from the install script in PEcAn. At some point I think we should see if we can minimize the number of dependencies and we can then also remove all the non ubuntu stuff, since the docker container will be only Ubuntu based.